src/main/java/de/tum/in/www1/artemis/domain/quiz/QuizQuestionStatisticComponent.java

```diff
@@ -0,0 +1,21 @@
+package de.tum.in.www1.artemis.domain.quiz;
+
+public interface QuizQuestionStatisticComponent<T1 extends QuizQuestionStatistic, T2 extends QuizQuestionComponent<T3>, T3 extends QuizQuestion> {
```

Comment:
```suggestion
public interface QuizQuestionStatisticComponent<S extends QuizQuestionStatistic, C extends QuizQuestionComponent<T3>, Q extends QuizQuestion> {
```
I think the enumeration is unnecessary and makes it more complicated.

Reply:
thanks